<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Delvian Valentine</author>
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New RSA Keys</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>18</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>18</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Public key</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_pub_key">
     <property name="text">
      <string>Browse…</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Private key</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_priv_key">
     <property name="text">
      <string>Browse…</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_pub_key</tabstop>
  <tabstop>button_priv_key</tabstop>
  <tabstop>button_save</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_save</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
